README.md

```diff
@@ -1,3 +1,42 @@
 # did-resolver-api
-This project implements a DID resolving REST api for evan.network. 
-It allows you to query the evan.network for DID documents, providing a valid evan DID.
+This project implements a DID resolving REST api for evan.network.
+<br>It allows you to query the evan.network for DID documents, providing a valid evan DID.
+<br>In order to receive a DID, you have to send a `GET` request to the resolver endpoint at
+
+`https://agents.test.evan.network/api/smart-agents/smart-agent-did-resolver/did/get/:did`
```

Comment:
can we discuss the URL?
Since it's a new service I would like to to it correct from the beginning.

Suggestion based on https://apiguide.readthedocs.io/en/latest/build_and_publish/use_RESTful_urls.html

`https://agents.test.evan.network/api/did/:did`... also I fear the `smart-agents/smart-agent-did-resolver` part is needed bc of Action-hero ... :/

Reply:
Action endpoints could also be reduce to : `.../api/smart-agents/did-resolver/...`